plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/Main.java

```diff
@@ -20,9 +21,19 @@ public static void main(String[] args) throws IOException {
             throw new IOException("Failed to read command-line arguments", e);
         }
 
+
+        if (options.isShowVersion()) {
+            try {
+                options.showVersion();
+            } catch (IOException e) {
+                System.exit(1);
```

Comment:
Pass this on down the line...
```suggestion
                throw e;
```

Reply:
On the Gitter chat Tim said "the CLI should exit with a non-zero code if there's an error", which I interpreted to mean that it should be explicit. But I think Martin said that it should just exit by throwing an error, so I'm still a little confused about this.

Reply:
let me test it and see what it looks like both ways